manifests/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Similar to the comment above, I'd put these in `deploy/base` (or `deploy/kubernetes/base`).